### ISSUE TITLE
GUI's broken at file choosing page

### ISSUE THREAD

**Issue by Danimal-63** [BitsByWill/AutoGradr#1]
When I choose a file the GUI becomes weird

**Comment by BitsByWill** [comment]
Yeah, I just realized there is an issue in the declaration of the image icon

**Comment by Danimal-63** [comment]
I redownloaded the files and ran it again, it has the same problem, I'm thinking of just writing my own at this point. 7.2/10

**Comment by BitsByWill** [comment]
I just fixed it
